### ISSUE TITLE
pandoc document conversion failed with error 25

### ISSUE THREAD

**Issue by pdiakumis** [umccr/umccrise#84]
Getting this when using the `bcbio_v1.2.4` branch on Gadi:

```
Error reading bibliography file bibliography.bib:
(line 190, column 15):
unexpected 'e'
Error: pandoc document conversion failed with error 25
Execution halted
```

Error is due to the newer pandoc v2.17.1.1 finding a weird character in PCGR's `bibliography.bib`. This was actually solved in PCGR v1.0.0 (see [commit](https://github.com/sigven/pcgr/pull/151/commits/fe921d590c6482c4d745b86432dd185fe0746c2a) from [this PR](https://github.com/sigven/pcgr/pull/151)).
Pinning pandoc to v2.16.1 in the `umccrise_pcgr` conda env solves this.

**Comment by scwatts** [comment]
Relevant [commit](https://github.com/umccr/umccrise/pull/78/commits/1ccfdaf673cfb0c0bf875f28dfd20fb758af6372) - change present on default branch and not on the bcbio branch as iirc the issue was observed only alongside the updates for DRAGEN.

Possible that some changes to Conda channels/preferred package versions have occurred, triggering this problem and the other you've reported related to MultiQC.

**Comment by pdiakumis** [comment]
All good, just winging it with this branch for the next X weeks/months until Gadi gets retired.

**Comment by pdiakumis** [comment]
Solved via #86
